packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -186,10 +236,10 @@ const useSandbox = (
     () =>
       createContainerMessageListener(
         {
-          setContent,
+          setContent: onUpdate,
           setPluginReady: onLoaded,
-          setHeight,
-          setModalOpen: setModalOpen,
+          setHeight: onHeightChange,
+          setModalOpen: onModalChange,
```

Comment:
The same as the previous comment:

![image](https://github.com/storyblok/field-plugin/assets/1240591/1829124e-5b3c-4b1d-9ba5-20861797048f)